### PR TITLE
test: Atomic updates

### PR DESCRIPTION
**Pull request by dperpeet**
follow-up to #5019

### CODE REVIEW

**Comment by dperpeet** [comment]
bot: Image refresh for rhel-atomic

**Comment by cockpituous** [comment]
Image creation for rhel-atomic in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-rhel-atomic-7aef13c0034556bdea61969bd545c5809bedd2ac/

**Comment by cockpituous** [comment]
Image creation failed.

**Comment by petervo** [comment]
When there are no upgrades to install

```
checkout=$(ostree refs | head -n 1)
```

when we do upgrade

```
checkout="$remote:$ref"
```

then we can

```
ostree checkout "$checkout" /var/local-tree
```

File: test/images/scripts/atomic.bootstrap [hunk 1]
```diff
@@ -22,15 +22,20 @@ set -e
...
-# download the newest base image
-if [[ "$prefix" == *qcow2 ]]; then
-    pattern="$prefix"
-else
-    pattern="$prefix-[0-9\.]+.x86_64.qcow2"
-fi
-newest=`wget -q -O- $base | egrep -o $pattern | sort -V | tail -1`
+# Lookup the newest base image recursively
+url=$base
+while [ $# -gt 2 ]; do
```
Comment (petervo) [inline test/images/scripts/atomic.bootstrap]: did you test this change with continuous as well?

**Comment by dperpeet** [comment]
bot: Image refresh for rhel-atomic

**Comment by cockpituous** [comment]
Image creation for rhel-atomic in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-rhel-atomic-76799e298c2753adc5e9a49427d683e88f32f842/

**Comment by dperpeet** [comment]
bot: Image refresh for rhel-atomic

**Comment by cockpituous** [comment]
Image creation for rhel-atomic in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-rhel-atomic-76799e298c2753adc5e9a49427d683e88f32f842/

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by dperpeet** [comment]
bot: Image refresh for rhel-atomic

**Comment by cockpituous** [comment]
Image creation for rhel-atomic in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-rhel-atomic-4ca2136320dc5f502ad2a3f5c9ee40713ce8e834/

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by cockpituous** [comment]
Image creation aborted

File: test/images/scripts/atomic.bootstrap (hunk 1 above)
Comment (dperpeet) [inline test/images/scripts/atomic.bootstrap]: I created the image successfully and uploaded.

**Comment by dperpeet** [comment]
kubernetes dashboard test has failed three times now with an error that looks the same:
```
Traceback (most recent call last):
  File "/build/cockpit/test/verify/kubelib.py", line 403, in testDashboard
    b.wait_text("#service-list tr[data-name='mock']:first-of-type td.containers", "1")
  File "/build/cockpit/test/common/testlib.py", line 247, in wait_text
    return self.wait_js_func('ph_text_is', selector, text)
  File "/build/cockpit/test/common/testlib.py", line 211, in wait_js_func
    return self.phantom.wait("%s(%s)" % (func, ','.join(map(jsquote, args))))
  File "/build/cockpit/test/common/testlib.py", line 688, in <lambda>
    return lambda *args: self._invoke(name, *args)
  File "/build/cockpit/test/common/testlib.py", line 711, in _invoke
    raise Error(res['error'])
Error: timeout
```

**Comment by dperpeet** [comment]
adding priority label because we need this in order to update our atomic test images again

**Comment by stefwalter** [comment]
Retriggering because I've seen the failure before.

**Pull request closed without merging**